.github/workflows/website.yml

```diff
@@ -14,6 +14,9 @@ jobs:
     steps:
       - uses: actions/checkout@v1
 
+      - name: Install SASS
+        run: npm install -g sass
+
```

Comment:
The CI failed in https://github.com/perlang-org/perlang/pull/179/checks?check_run_id=2308969743; this change is here to try and fix that. I don't really understand _why_ it tried to rebuild the CSS stylesheet though, but I'm guessing the timestamp of the `.scss` file was never than the `.css` file in this case. It has worked until now, probably because the order of the timestamps was the reverse in all CI runs until this one.